### ISSUE TITLE
[Bug] Process Memory Provider can't open any process

### ISSUE THREAD

**Issue by HikariXP** [WerWolv/ImHex#1808]
### Operating System

Windows

### What's the issue you encountered?

When i use process memory provider to open any process, it will fill with question mark and zero. Have no idea is there any step i missed?
![image-20240712110559751](https://github.com/user-attachments/assets/4e7626ee-8bc9-4659-9c95-f8b34ecb81ec)

### How can the issue be reproduced?

ImHex use process memory provider open any process

### ImHex Version

1.35.3

### ImHex Build Type

- [ ] Nightly or built from sources

### Installation type

MSI

**Comment by Surasia** [comment]
You'll have to find the base adress of the process, through something like x64dbg.

**Comment by paxcut** [comment]
Is getting the base address assigned by the OS when the process is created automatically something very hard to do?
Is that base address the only one that makes sense? I guess it can always be changed but it would be a lot less confusing if it loaded it for you when you first open the process.

**Comment by WerWolv** [comment]
It's pretty normal that there's no data available at address 0x00 :) 
ImHex shows the entire 64 bit address space of your process which, for the most part, will be unmapped data (denoted with the `??` bytes you're seeing). Click on the little Provider Settings button on the left side of the screen and you'll get a list of all the regions that have been mapped. 
![image](https://github.com/user-attachments/assets/3e8a1a26-0d14-4978-a31b-f322a09f4845)
Clicking on one of them then will bring you to that data.

**Comment by HikariXP** [comment]
First of all, I'm sorry that I'm a novice in this area, and I'm sorry for the problems caused by some obvious operations that I missed.
I followed Surasia's advice and used x64dbg to get the Base Address. Then I got it through the Process Memory Provider and set the base address. Only 00 and ??
I just realized that maybe I should go to the forum and ask about it first. It's not appropriate to post it in an issue
